archive/g/go/rot-13.go

```diff
@@ -0,0 +1,34 @@
+package main
+
+import (
+	"fmt"
+	"os"
+	"strings"
+)
+
+func rot13(str string) string {
+	return strings.Map(encryptRune, str)
```

Comment:
Really clean functional code here! 👍 beats the hell out of looping over the terms yourself.